02.general/17.dns-providers/docs.en.md

```diff
@@ -428,3 +433,85 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
+| DoT, IPv4 | Provider: `Surfnet` IP: `145.100.185.16` Hostname: `dnsovertls1.sinodun.com` | <a href="sdns://AwAAAAAAAAAAAAAddGxzOi8vZG5zb3ZlcnRsczEuc2lub2R1bi5jb20">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `Surfnet` IP: `2001:610:1:40ba:145:100:185:16` Hostname `dnsovertls1.sinodun.com` | <a href="sdns://AwAAAAAAAAAAAAAddGxzOi8vZG5zb3ZlcnRsczEuc2lub2R1bi5jb20">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `getdnsapi.net` IP: `185.49.141.37` Hostname: `getdnsapi.net` | <a href="sdns://AwAAAAAAAAAAAAATdGxzOi8vZ2V0ZG5zYXBpLm5ldA">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `getdnsapi.net` IP: `2a04:b900:0:100::37` Hostname `getdnsapi.net` | <a href="sdns://AwAAAAAAAAAAAAATdGxzOi8vZ2V0ZG5zYXBpLm5ldA">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `UncensoredDNS` IP: `89.233.43.71 ` Hostname: `unicast.censurfridns.dk` | <a href="sdns://AwAAAAAAAAAAAAAddGxzOi8vdW5pY2FzdC5jZW5zdXJmcmlkbnMuZGs">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `UncensoredDNS` IP: `2a01:3a0:53:53::0` Hostname `unicast.censurfridns.dk` | <a href="sdns://AwAAAAAAAAAAAAAddGxzOi8vdW5pY2FzdC5jZW5zdXJmcmlkbnMuZGs">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `Fondation RESTENA ` IP: `158.64.1.29` Hostname: `kaitain.restena.lu` | <a href="sdns://AwAAAAAAAAAAAAAYdGxzOi8va2FpdGFpbi5yZXN0ZW5hLmx1">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `Fondation RESTENA ` IP: `2001:a18:1::29` Hostname `kaitain.restena.lu` | <a href="sdns://AwAAAAAAAAAAAAAYdGxzOi8va2FpdGFpbi5yZXN0ZW5hLmx1">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `dkg` IP: `199.58.81.218` Hostname: `dns.cmrg.net` | <a href="sdns://AwAAAAAAAAAAAAASdGxzOi8vZG5zLmNtcmcubmV0">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `dkg` IP: `2001:470:1c:76d::53` Hostname `dns.cmrg.net` | <a href="sdns://AwAAAAAAAAAAAAASdGxzOi8vZG5zLmNtcmcubmV0t">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `dns.larsdebruin.net` IP: `51.15.70.167` Hostname: `dns.larsdebruin.net` | <a href="sdns://AwAAAAAAAAAAAAAZdGxzOi8vZG5zLmxhcnNkZWJydWluLm5ldA">Add to AdGuard</a> |
+| DoT, IPv4 | Provider: `dns-tls.bitwiseshift.net` IP: `81.187.221.24` Hostname: `dns-tls.bitwiseshift.net` | <a href="sdns://AwAAAAAAAAAAAAAedGxzOi8vZG5zLXRscy5iaXR3aXNlc2hpZnQubmV0">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `dns-tls.bitwiseshift.net` IP: `2001:8b0:24:24::24` Hostname `dns-tls.bitwiseshift.net` | <a href="sdns://AwAAAAAAAAAAAAAedGxzOi8vZG5zLXRscy5iaXR3aXNlc2hpZnQubmV0">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `ns1.dnsprivacy.at` IP: `94.130.110.185` Hostname: `ns1.dnsprivacy.at` | <a href="sdns://AwAAAAAAAAAAAAAXdGxzOi8vbnMxLmRuc3ByaXZhY3kuYXQ">Add to AdGuard</a> |
+| DoT, IPv6 | Provider: `ns1.dnsprivacy.at` IP: `2a01:4f8:c0c:3c03::2` Hostname `ns1.dnsprivacy.at` | <a href="sdns://AwAAAAAAAAAAAAAXdGxzOi8vbnMxLmRuc3ByaXZhY3kuYXQ">Add to AdGuard</a>  |
+| DoT, IPv4 | Provider: `ns2.dnsprivacy.at` IP: `2a01:4f8:c0c:3bfc::2` Hostname: `ns2.dnsprivacy.at` | <a href="sdns://AwAAAAAAAAAAAAAXdGxzOi8vbnMyLmRuc3ByaXZhY3kuYXQ">Add to AdGuard</a> |
```

Comment:
2a01:4f8:c0c:3bfc::2 is not an IPv4 addr :)

Reply:
hi @admitrevskiy take a look on my changes :)